remote/gitea/gitea_oauth.go

```diff
@@ -0,0 +1,389 @@
+// Copyright 2018 Drone.IO Inc.
```

Comment:
Where is this file coming from? The license is fine, but best to trace the origin.. just in case.

Can you comment a link with the origin?

Reply:
It's mostly copy/paste from gitea.go in the same directory.